diff -Naur ams-2.0.1/src/main.cpp ams/src/main.cpp
--- ams-2.0.1/src/main.cpp  2009-06-12 23:22:19.000000000 +0200
+++ ams/src/main.cpp    2009-12-28 18:10:58.000000000 +0100
@@ -61,8 +61,8 @@
     QDir amshome = QDir(QDir::homePath());
     if (!amshome.exists(AMSDIR)) {
         if (!amshome.mkdir(AMSDIR)) {
-            qWarning(QObject::tr("Could not create ams home "
-                        "directory.").toUtf8());
+            qWarning("%s", QObject::tr("Could not create ams home "
+                        "directory.").toLatin1().constData());
             return -1;
         }
     }
@@ -72,8 +72,8 @@
         //StdOut << "Resource file path: " << rcPath << endl;
         fd = open(rcPath.toLatin1().data(), O_CREAT|O_RDWR, 0666);
         if (fd == -1) {
-            qWarning(QObject::tr("Failed to open file '%1'")
-                    .arg(rcPath).toUtf8());
+            qWarning("%s", QObject::tr("Failed to open file '%1'")
+                    .arg(rcPath).toLatin1().constData());
             return -1;
         }
 
@@ -84,15 +84,16 @@
         } else {
             lock.l_type = F_RDLCK;
             if (fcntl(fd, F_SETLK, &lock) == -1) {
-                qWarning(QObject::tr("Ooops in %1 at %2")
-                        .arg(__FUNCTION__).arg(__LINE__).toUtf8());
+                qWarning("%s", QObject::tr("Ooops in %1 at %2")
+                        .arg(__FUNCTION__).arg(__LINE__).toLatin1().constData());
                 return -1;
             }
             name = amsSynthName(name, index);
             return fd;
         }
     }
-    qWarning(QObject::tr("Client name '%1' occupied.").arg(name).toUtf8());
+    qWarning("%s", QObject::tr("Client name '%1' occupied.").arg(name)
+            .toLatin1().constData());
     return -1;
 }
 
@@ -118,7 +119,7 @@
       app.installTranslator(&qtTr);
   else
       qWarning("No Qt translation for locale '%s' found.",
-              loc.name().toUtf8().constData());
+              loc.name().toLatin1().constData());
   
   // translator for ams strings
   QTranslator amsTr;
@@ -127,7 +128,7 @@
       app.installTranslator(&amsTr);
   else
       qWarning("No " AMS_LONGNAME " translation for locale '%s' found.",
-              loc.name().toUtf8().constData());
+              loc.name().toLatin1().constData());
   
   int getopt_return;
   int option_index;
diff -Naur ams-2.0.1/src/mainwindow.cpp ams/src/mainwindow.cpp
--- ams-2.0.1/src/mainwindow.cpp    2009-05-25 10:22:32.000000000 +0200
+++ ams/src/mainwindow.cpp  2009-12-28 18:10:58.000000000 +0100
@@ -171,14 +171,16 @@
   updateWindowTitle();
 
   if (mso.havePresetPath) {
-    qWarning(QObject::tr("Preset path now %1").arg(mso.presetPath).toUtf8()); 
+    qWarning("%s", QObject::tr("Preset path now %1").arg(mso.presetPath)
+            .toUtf8().constData()); 
     modularSynth->setLoadPath(mso.presetPath);
   }
   modularSynth->go(mso.forceJack, mso.forceAlsa);
 
   // autoload patch file
   if (mso.havePreset) {
-    qWarning(QObject::tr("Loading preset %1").arg(mso.presetName).toUtf8()); 
+    qWarning("%s", QObject::tr("Loading preset %1").arg(mso.presetName)
+            .toUtf8().constData()); 
     openFile(mso.presetName);
   }
 
@@ -191,21 +193,27 @@
 
 MainWindow::~MainWindow()
 {
-    qWarning(QObject::tr("Closing synthesizer...").toUtf8());
+    qWarning("%s", QObject::tr("Closing synthesizer...").toUtf8().constData());
     writeConfig();
 
     // remove file lock
     struct flock lock = {F_WRLCK, SEEK_SET, 0, 0, 0};
     if (fcntl(rcFd, F_UNLCK, &lock) == -1) {
-        qWarning(QObject::tr("Could not unlock preferences file.").toUtf8());
+        qWarning("%s", QObject::tr("Could not unlock preferences file.")
+                .toUtf8().constData());
     }
 }
 
 
 void MainWindow::sighandler(int s)
 {
-  char pipeMessage = s;
-  write(pipeFd[1], &pipeMessage, 1);
+    ssize_t result;
+
+    char pipeMessage = s;
+    result = write(pipeFd[1], &pipeMessage, 1);
+    if (result == -1)
+        qWarning("%s", QObject::tr("Error writing to pipe: %1")
+                .arg(errno).toUtf8().constData());
 }
 
 void MainWindow::unixSignal(int /*fd*/)
@@ -303,7 +311,8 @@
     QFile f(fn);
 
     if (!f.open(QIODevice::ReadOnly)) {
-        qWarning(tr("Could not read file '%1'").arg(fn).toUtf8());
+        qWarning("%s", tr("Could not read file '%1'").arg(fn)
+                .toUtf8().constData());
         return;
     }
 
@@ -334,7 +343,8 @@
 
     QFile f(fileName);
     if (!f.open(QIODevice::WriteOnly)) {
-        qWarning(tr("Could not open file '%1'").arg(fileName).toUtf8());
+        qWarning("%s", tr("Could not open file '%1'").arg(fileName)
+                .toUtf8().constData());
         return false;
     }
 
@@ -365,7 +375,7 @@
         saveFile();
     }
     else
-        qWarning(tr("Saving aborted").toUtf8());
+        qWarning("%s", tr("Saving aborted").toUtf8().constData());
 }
 
 
@@ -399,11 +409,11 @@
     QFile file;
 
     if (!file.open(rcFd, QIODevice::ReadOnly)) {
-        qWarning("Could not open preferences file.");
+        qWarning("%s", "Could not open preferences file.");
         return;
     }
     if (!file.seek(0)) {
-        qWarning("Could not seek start of preferences file.");
+        qWarning("%s", "Could not seek start of preferences file.");
         file.close();
         return;
     }
@@ -425,12 +435,12 @@
     QFile file;
 
     if (!file.open(rcFd, QIODevice::WriteOnly)) {
-        qWarning("Could not open preferences file.");
+        qWarning("%s", "Could not open preferences file.");
         return;
     }
 
     if (!file.resize(0)) {
-        qWarning("Could not resize preferences file.");
+        qWarning("%s", "Could not resize preferences file.");
         file.close();
         return;
     }
diff -Naur ams-2.0.1/src/modularsynth.cpp ams/src/modularsynth.cpp
--- ams-2.0.1/src/modularsynth.cpp  2009-06-12 23:07:41.000000000 +0200
+++ ams/src/modularsynth.cpp    2009-12-28 18:10:58.000000000 +0100
@@ -281,7 +281,7 @@
     if ((synthdata->seq_handle = open_seq()))
         initSeqNotifier();
     else
-        qWarning(QObject::tr("Alsa MIDI wont work!").toUtf8());
+        qWarning("%s", QObject::tr("Alsa MIDI wont work!").toUtf8().constData());
 
     midiWidget->setActiveMidiControllers();
 
@@ -371,7 +371,7 @@
 
     if (snd_seq_open(&seq_handle, "hw", SND_SEQ_OPEN_DUPLEX,
                 SND_SEQ_NONBLOCK) < 0) {
-        qWarning(QObject::tr("Error opening ALSA sequencer.").toUtf8());
+        qWarning("%s", QObject::tr("Error opening ALSA sequencer.").toUtf8().constData());
         return NULL;
     }
 
@@ -381,7 +381,7 @@
     if (snd_seq_create_simple_port(seq_handle, "ams in",
                 SND_SEQ_PORT_CAP_WRITE | SND_SEQ_PORT_CAP_SUBS_WRITE,
                 SND_SEQ_PORT_TYPE_APPLICATION) < 0) {
-        qWarning(QObject::tr("Error creating sequencer write port.").toUtf8());
+        qWarning("%s", QObject::tr("Error creating sequencer write port.").toUtf8().constData());
         snd_seq_close(seq_handle);
         return NULL;
     }
@@ -390,7 +390,7 @@
                     snd_seq_create_simple_port(seq_handle, "ams out",
                         SND_SEQ_PORT_CAP_READ | SND_SEQ_PORT_CAP_SUBS_READ,
                         SND_SEQ_PORT_TYPE_APPLICATION)) < 0) {
-            qWarning(QObject::tr("Error creating sequencer read port.").toUtf8());
+            qWarning("%s", QObject::tr("Error creating sequencer read port.").toUtf8().constData());
             snd_seq_close(seq_handle);
             return NULL;
         }
@@ -950,7 +950,7 @@
         }
     } 
     else
-        qWarning(QObject::tr("All available output ports are in use").toUtf8());
+        qWarning("%s", QObject::tr("All available output ports are in use").toUtf8().constData());
 }
 
 void ModularSynth::newM_pcmin()
@@ -968,7 +968,7 @@
         }
     } 
     else
-        qWarning(QObject::tr("All available input ports are in use").toUtf8());
+        qWarning("%s", QObject::tr("All available input ports are in use").toUtf8().constData());
 }
 
 void ModularSynth::newM_scope() {
@@ -1254,7 +1254,7 @@
               newLadspaPolyFlag = tokens[5].toInt();
               ladspaLibName = tokens[6];
               pluginName = qs.section(' ', 7);
-              qWarning(tr("Loading LADSPA plugin \"%1\" from library \"%2\".") 
+              qWarning("%s", tr("Loading LADSPA plugin \"%1\" from library \"%2\".") 
                       .arg(pluginName).arg(ladspaLibName).toUtf8().constData());
 
               if (!synthdata->getLadspaIDs(ladspaLibName, pluginName,
@@ -1477,9 +1477,9 @@
             continue;
 
         if (inport->hasConnectedPort()) {
-            qWarning(tr("Input port %1 of module %2 is already connected. "
+            qWarning("%s", tr("Input port %1 of module %2 is already connected. "
                         "New connection to module %3 ignored.")
-                    .arg(index1).arg(moduleID1).arg(moduleID2).toUtf8());
+                    .arg(index1).arg(moduleID1).arg(moduleID2).toUtf8().constData());
             continue;
         }
 
@@ -1734,8 +1734,8 @@
             }
         }
         else
-            qWarning(tr("Unknown MIDI controller tag found: %1")
-                    .arg(qs).toUtf8());
+            qWarning("%s", tr("Unknown MIDI controller tag found: %1")
+                    .arg(qs).toUtf8().constData());
     }
 
     // #PARA# <te_id> <..> <line_idx>
@@ -1770,8 +1770,8 @@
         if (pos != -1)
             guiWidget->addTab(rx.cap(1));
         else
-            qWarning(QObject::tr("No title for tab '%1' found.")
-                    .arg(qs).toUtf8());
+            qWarning("%s", QObject::tr("No title for tab '%1' found.")
+                    .arg(qs).toUtf8().constData());
     }
 
     // Frame <"FrameName"> <frame number>
@@ -1786,8 +1786,8 @@
             guiWidget->addFrame(rx.cap(1));
         }
         else
-            qWarning(QObject::tr("No data for frame '%1' found.")
-                    .arg(qs).toUtf8());
+            qWarning("%s", QObject::tr("No data for frame '%1' found.")
+                    .arg(qs).toUtf8().constData());
     }
 
     // Parameter <"ParamName"> <modid> <index>
@@ -1801,7 +1801,7 @@
             numbers = numbers.trimmed();
             tokens = numbers.split(' ');
             if (tokens.isEmpty()) {
-                qWarning(QObject::tr("No parameter values found.").toUtf8());
+                qWarning("%s", QObject::tr("No parameter values found.").toUtf8().constData());
                 continue;
             }
 
@@ -1810,8 +1810,8 @@
             index = tokens[1].toInt();
         }
         else {
-            qWarning(QObject::tr("No parameter name '%1' found.")
-                    .arg(qs).toUtf8());
+            qWarning("%s", QObject::tr("No parameter name '%1' found.")
+                    .arg(qs).toUtf8().constData());
             continue;
         }
         m = getModuleWithId(moduleID);
@@ -1876,8 +1876,8 @@
         if (pos != -1)
             qs = rx.cap(1);
         else {
-            qWarning(QObject::tr("No name for preset '%1' found.")
-                    .arg(qs).toUtf8());
+            qWarning("%s", QObject::tr("No name for preset '%1' found.")
+                    .arg(qs).toUtf8().constData());
             continue;
         }
 
@@ -1886,7 +1886,7 @@
     }
     /*For debugging only, 'Module' and 'Comment' will also apear
     else
-        qWarning(tr("Unknown tag found: %1").arg(qs).toUtf8());
+        qWarning("%s", tr("Unknown tag found: %1").arg(qs).toUtf8().constData());
     */
   } // end while loop
 
diff -Naur ams-2.0.1/src/m_wavout.cpp ams/src/m_wavout.cpp
--- ams-2.0.1/src/m_wavout.cpp  2009-06-09 09:50:30.000000000 +0200
+++ ams/src/m_wavout.cpp    2009-12-28 18:10:58.000000000 +0100
@@ -235,15 +235,20 @@
 
 void M_wavout::openBrowser() {
 
-  char buf[2048];
+    char buf[2048];
+    char * result;
 
-  getcwd(buf, 2048);
-  wavname = QFileDialog::getSaveFileName(this, tr("Choose Wave File"),
-          buf, tr("WAV files (*.wav)"));
-  if (!wavname.isEmpty()) {
-    configDialog->lineEditList.at(0)->setText(wavname);    
-    createWav();
-  }
+    result = getcwd(buf, 2048);
+    if (result == NULL) {
+        qWarning("Error in getcwd((): %d", errno);
+        return;
+    }
+    wavname = QFileDialog::getSaveFileName(this, tr("Choose Wave File"),
+            buf, tr("WAV files (*.wav)"));
+    if (!wavname.isEmpty()) {
+        configDialog->lineEditList.at(0)->setText(wavname);    
+        createWav();
+    }
 }
 
 void M_wavout::timerProc() {          
diff -Naur ams-2.0.1/src/synthdata.cpp ams/src/synthdata.cpp
--- ams-2.0.1/src/synthdata.cpp	2009-09-26 19:27:53.000000000 +0200
+++ ams/src/synthdata.cpp	2009-12-28 18:10:58.000000000 +0100
@@ -389,13 +389,14 @@
     pthread_attr_setinheritsched (&attr, PTHREAD_EXPLICIT_SCHED);
     if (pthread_create (&alsa_thread, &attr, alsa_static_thr_main, this))
     {
-        qWarning(QObject::tr("Can't create ALSA thread with RT priority").toUtf8());
+        qWarning("%s", QObject::tr("Can't create ALSA thread with RT priority")
+                .toUtf8().constData());
         pthread_attr_setschedpolicy (&attr, SCHED_OTHER);
         parm.sched_priority = sched_get_priority_max (SCHED_OTHER);
         pthread_attr_setschedparam (&attr, &parm);
         if (pthread_create (&alsa_thread, &attr, alsa_static_thr_main, this))
         {
-            qWarning(QObject::tr("Can't create ALSA thread").toUtf8());
+            qWarning("%s", QObject::tr("Can't create ALSA thread").toUtf8().constData());
             exit (1);
 	}
     }
@@ -404,7 +405,7 @@
 
 int SynthData::closeAlsa ()
 {
-    qWarning(QObject::tr("Closing ALSA...").toUtf8());
+    qWarning("%s", QObject::tr("Closing ALSA...").toUtf8().constData());
     withAlsa = false;
     sleep (1);
     delete alsa_handle;
@@ -497,7 +498,7 @@
     jack_handle = jack_client_open(name.toLatin1().constData(),
 				   JackNullOption, NULL);
     if (!jack_handle) {
-        qWarning(QObject::tr("Can't connect to JACK").toUtf8());
+        qWarning("%s", QObject::tr("Can't connect to JACK").toUtf8().constData());
         return -ENODEV;
     }
 
@@ -520,7 +521,7 @@
 
     if (jack_activate (jack_handle))
     {
-        qWarning(QObject::tr("Can't activate JACK").toUtf8());
+        qWarning("%s", QObject::tr("Can't activate JACK").toUtf8().constData());
         exit (1);
     }
 
@@ -532,7 +533,7 @@
 
 int SynthData::closeJack ()
 {
-    qWarning(QObject::tr("Closing JACK...").toUtf8());
+    qWarning("%s", QObject::tr("Closing JACK...").toUtf8().constData());
     jack_deactivate (jack_handle);
     for (int i = 0; i < play_ports; i++)
         jack_port_unregister(jack_handle, jack_out[i]);
diff -Naur ams-2.0.1/src/translations/ams_de.ts ams/src/translations/ams_de.ts
--- ams-2.0.1/src/translations/ams_de.ts    2009-12-26 01:03:32.000000000 +0100
+++ ams/src/translations/ams_de.ts  2009-12-28 18:10:58.000000000 +0100
@@ -1,5 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
-<!DOCTYPE TS><TS version="1.1">
+<!DOCTYPE TS>
+<TS version="2.0">
 <context>
     <name>ColorWidget</name>
     <message>
@@ -138,11 +139,15 @@
         <translation>Stimme</translation>
     </message>
     <message>
+        <location filename="../m_ad.cpp" line="74"/>
+        <location filename="../m_ad.cpp" line="172"/>
         <location filename="../m_ad.cpp" line="176"/>
         <source>Detune %1</source>
         <translation>Verstimmung %1</translation>
     </message>
     <message>
+        <location filename="../m_ad.cpp" line="78"/>
+        <location filename="../m_ad.cpp" line="174"/>
         <location filename="../m_ad.cpp" line="178"/>
         <source>Drift %1</source>
         <translation type="unfinished"></translation>
@@ -334,6 +339,7 @@
         <translation type="unfinished"></translation>
     </message>
     <message>
+        <location filename="../m_advmcv.cpp" line="35"/>
         <location filename="../m_advmcv.cpp" line="61"/>
         <source>Controller %1</source>
         <translation type="unfinished"></translation>
@@ -416,6 +422,7 @@
 <context>
     <name>M_delay</name>
     <message>
+        <location filename="../m_delay.cpp" line="23"/>
         <location filename="../m_delay.cpp" line="33"/>
         <source>Delay</source>
         <translation type="unfinished"></translation>
@@ -866,7 +873,7 @@
         <translation type="unfinished"></translation>
     </message>
     <message>
-        <location filename="../m_mphlfo.cpp" line="49"/>
+        <location filename="../m_mphlfo.cpp" line="48"/>
         <source>Saw Down</source>
         <translation type="unfinished"></translation>
     </message>
@@ -1008,6 +1015,7 @@
         <translation type="unfinished"></translation>
     </message>
     <message>
+        <location filename="../m_pcmout.cpp" line="33"/>
         <location filename="../m_pcmout.cpp" line="53"/>
         <source>Keep</source>
         <translation type="unfinished"></translation>
@@ -1182,6 +1190,7 @@
         <translation type="unfinished"></translation>
     </message>
     <message>
+        <location filename="../m_scmcv.cpp" line="67"/>
         <location filename="../m_scmcv.cpp" line="176"/>
         <source>   Scale: </source>
         <translation type="unfinished"></translation>
@@ -1210,6 +1219,7 @@
 <context>
     <name>M_scope</name>
     <message>
+        <location filename="../m_scope.cpp" line="28"/>
         <location filename="../m_scope.cpp" line="49"/>
         <source>Scope</source>
         <translation type="unfinished"></translation>
@@ -1318,6 +1328,7 @@
         <translation type="unfinished"></translation>
     </message>
     <message>
+        <location filename="../m_scquantizer.cpp" line="51"/>
         <location filename="../m_scquantizer.cpp" line="204"/>
         <source>   Scale: </source>
         <translation type="unfinished"></translation>
@@ -1341,6 +1352,7 @@
         <translation type="unfinished"></translation>
     </message>
     <message>
+        <location filename="../m_seq.cpp" line="33"/>
         <location filename="../m_seq.cpp" line="38"/>
         <source>Trigger</source>
         <translation type="unfinished"></translation>
@@ -1430,16 +1442,19 @@
 <context>
     <name>M_spectrum</name>
     <message>
+        <location filename="../m_spectrum.cpp" line="7"/>
         <location filename="../m_spectrum.cpp" line="88"/>
         <source>Spectrum</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
+        <location filename="../m_spectrum.cpp" line="13"/>
         <location filename="../m_spectrum.cpp" line="60"/>
         <source>In 0</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
+        <location filename="../m_spectrum.cpp" line="14"/>
         <location filename="../m_spectrum.cpp" line="61"/>
         <source>In 1</source>
         <translation type="unfinished"></translation>
@@ -1573,7 +1588,7 @@
 <context>
     <name>M_stereomix</name>
     <message>
-        <location filename="../m_stereomix.cpp" line="107"/>
+        <location filename="../m_stereomix.cpp" line="106"/>
         <source>Stereo Mixer %1</source>
         <translation type="unfinished"></translation>
     </message>
@@ -1626,16 +1641,19 @@
         <translation type="unfinished"></translation>
     </message>
     <message>
+        <location filename="../m_vca.cpp" line="33"/>
         <location filename="../m_vca.cpp" line="40"/>
         <source>Gain 1</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
+        <location filename="../m_vca.cpp" line="34"/>
         <location filename="../m_vca.cpp" line="41"/>
         <source>In 0</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
+        <location filename="../m_vca.cpp" line="35"/>
         <location filename="../m_vca.cpp" line="42"/>
         <source>In 1</source>
         <translation type="unfinished"></translation>
@@ -1968,6 +1986,7 @@
         <translation type="unfinished"></translation>
     </message>
     <message>
+        <location filename="../m_vcf.cpp" line="37"/>
         <location filename="../m_vcf.cpp" line="56"/>
         <source>Resonance</source>
         <translation type="unfinished"></translation>
@@ -2071,6 +2090,7 @@
         <translation type="unfinished"></translation>
     </message>
     <message>
+        <location filename="../m_vco.cpp" line="44"/>
         <location filename="../m_vco.cpp" line="61"/>
         <source>PW</source>
         <translation type="unfinished"></translation>
@@ -2086,6 +2106,7 @@
         <translation type="unfinished"></translation>
     </message>
     <message>
+        <location filename="../m_vco.cpp" line="49"/>
         <location filename="../m_vco.cpp" line="67"/>
         <source>Saw</source>
         <translation type="unfinished"></translation>
@@ -2194,6 +2215,7 @@
         <translation type="unfinished"></translation>
     </message>
     <message>
+        <location filename="../m_vco2.cpp" line="41"/>
         <location filename="../m_vco2.cpp" line="69"/>
         <source>PW</source>
         <translation type="unfinished"></translation>
@@ -2382,6 +2404,8 @@
         <translation type="unfinished"></translation>
     </message>
     <message>
+        <location filename="../m_vcorgan.cpp" line="59"/>
+        <location filename="../m_vcorgan.cpp" line="66"/>
         <location filename="../m_vcorgan.cpp" line="68"/>
         <source>Volume %1</source>
         <translation type="unfinished"></translation>
@@ -2569,12 +2593,12 @@
 <context>
     <name>M_wavout</name>
     <message>
-        <location filename="../m_wavout.cpp" line="241"/>
+        <location filename="../m_wavout.cpp" line="246"/>
         <source>Choose Wave File</source>
         <translation>Wave-Datei wählen</translation>
     </message>
     <message>
-        <location filename="../m_wavout.cpp" line="242"/>
+        <location filename="../m_wavout.cpp" line="247"/>
         <source>WAV files (*.wav)</source>
         <translation>WAV-Dateien (*.wav)</translation>
     </message>
@@ -2654,7 +2678,7 @@
         <translation type="unfinished"></translation>
     </message>
     <message>
-        <location filename="../m_wavout.cpp" line="272"/>
+        <location filename="../m_wavout.cpp" line="277"/>
         <source>Time: %1:%2:%3 </source>
         <translation type="unfinished"></translation>
     </message>
@@ -2687,6 +2711,7 @@
         <translation>&amp;Hilfe</translation>
     </message>
     <message>
+        <location filename="../mainwindow.cpp" line="49"/>
         <location filename="../mainwindow.cpp" line="52"/>
         <source>&amp;New</source>
         <translation>&amp;Neu</translation>
@@ -3052,71 +3077,72 @@
         <translation>&amp;Über AlsaModularSynth...</translation>
     </message>
     <message>
-        <location filename="../mainwindow.cpp" line="231"/>
+        <location filename="../mainwindow.cpp" line="239"/>
         <source>Unnamed file was changed.
 Save changes?</source>
         <translation>Die unbenannte Datei wurde geändert.
 Änderungen speichern?</translation>
     </message>
     <message>
-        <location filename="../mainwindow.cpp" line="234"/>
+        <location filename="../mainwindow.cpp" line="241"/>
         <source>File &apos;%1&apos; was changed.
 Save changes?</source>
         <translation>Die Datei &apos;%1&apos; wurde geändert.
 Änderungen speichern?</translation>
     </message>
     <message>
-        <location filename="../mainwindow.cpp" line="236"/>
+        <location filename="../mainwindow.cpp" line="244"/>
         <source>Save changes</source>
         <translation>Änderungen speichern</translation>
     </message>
     <message>
-        <location filename="../mainwindow.cpp" line="237"/>
+        <location filename="../mainwindow.cpp" line="245"/>
         <source>&amp;Yes</source>
         <translation>&amp;Ja</translation>
     </message>
     <message>
-        <location filename="../mainwindow.cpp" line="237"/>
+        <location filename="../mainwindow.cpp" line="245"/>
         <source>&amp;No</source>
         <translation>&amp;Nein</translation>
     </message>
     <message>
-        <location filename="../mainwindow.cpp" line="237"/>
+        <location filename="../mainwindow.cpp" line="245"/>
         <source>Cancel</source>
         <translation>Abbrechen</translation>
     </message>
     <message>
-        <location filename="../mainwindow.cpp" line="244"/>
+        <location filename="../mainwindow.cpp" line="252"/>
         <source>Open patch file</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
-        <location filename="../mainwindow.cpp" line="355"/>
+        <location filename="../mainwindow.cpp" line="253"/>
+        <location filename="../mainwindow.cpp" line="365"/>
         <source>Patch files</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
-        <location filename="../mainwindow.cpp" line="306"/>
+        <location filename="../mainwindow.cpp" line="314"/>
         <source>Could not read file &apos;%1&apos;</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
-        <location filename="../mainwindow.cpp" line="337"/>
+        <location filename="../mainwindow.cpp" line="346"/>
         <source>Could not open file &apos;%1&apos;</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
-        <location filename="../mainwindow.cpp" line="354"/>
+        <location filename="../mainwindow.cpp" line="364"/>
         <source>Save patch file</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
-        <location filename="../mainwindow.cpp" line="368"/>
+        <location filename="../mainwindow.cpp" line="378"/>
         <source>Saving aborted</source>
         <translation>Speichern abgebrochen</translation>
     </message>
     <message>
-        <location filename="../mainwindow.cpp" line="377"/>
+        <location filename="../mainwindow.cpp" line="387"/>
         <source>noname</source>
         <translation>unbenannt</translation>
     </message>
@@ -3126,7 +3152,7 @@
         <translation>Über &amp;Qt...</translation>
     </message>
     <message>
-        <location filename="../mainwindow.cpp" line="393"/>
+        <location filename="../mainwindow.cpp" line="403"/>
         <source>About Qt</source>
         <translation>Über Qt</translation>
     </message>
@@ -3149,7 +3175,7 @@
 <context>
     <name>MidiControllerModel</name>
     <message>
-        <location filename="../midiwidget.cpp" line="78"/>
+        <location filename="../midiwidget.cpp" line="77"/>
         <source>Module</source>
         <translation>Modul</translation>
     </message>
@@ -3159,7 +3185,7 @@
         <translation type="unfinished"></translation>
     </message>
     <message>
-        <location filename="../midiwidget.cpp" line="65"/>
+        <location filename="../midiwidget.cpp" line="64"/>
         <source>Type: %1 Channel: %2 Param: %3</source>
         <translation type="unfinished"></translation>
     </message>
@@ -3237,7 +3263,7 @@
         <translation type="unfinished"></translation>
     </message>
     <message>
-        <location filename="../midiwidget.cpp" line="512"/>
+        <location filename="../midiwidget.cpp" line="511"/>
         <source>This will erase all presets for this configuration. Continue?</source>
         <translation type="unfinished"></translation>
     </message>
@@ -3290,7 +3316,6 @@
         <translation>Farben lasen</translation>
     </message>
     <message>
-        <location filename="../modularsynth.cpp" line="1168"/>
         <source>AlsaModularSynth Color files (*.acs)</source>
         <translation type="obsolete">Farbschemadateien (*.acs)</translation>
     </message>
@@ -3310,12 +3335,12 @@
         <translation>Konnte Datei nicht speichern.</translation>
     </message>
     <message>
-        <location filename="../modularsynth.cpp" line="1265"/>
+        <location filename="../modularsynth.cpp" line="1263"/>
         <source>Could not find LADSPA plugin &quot;%1&quot; from library &quot;%2&quot;.</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
-        <location filename="../modularsynth.cpp" line="1738"/>
+        <location filename="../modularsynth.cpp" line="1737"/>
         <source>Unknown MIDI controller tag found: %1</source>
         <translation type="unfinished"></translation>
     </message>
@@ -3325,17 +3350,18 @@
         <translation>Über </translation>
     </message>
     <message>
-        <location filename="../modularsynth.cpp" line="1482"/>
+        <location filename="../modularsynth.cpp" line="1480"/>
         <source>Input port %1 of module %2 is already connected. New connection to module %3 ignored.</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
-        <location filename="../modularsynth.cpp" line="1258"/>
+        <location filename="../modularsynth.cpp" line="1257"/>
         <source>Loading LADSPA plugin &quot;%1&quot; from library &quot;%2&quot;.</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
-        <location filename="../modularsynth.cpp" line="1168"/>
+        <location filename="../modularsynth.cpp" line="1125"/>
+        <location filename="../modularsynth.cpp" line="1167"/>
         <source>AlsaModularSynth Color files</source>
         <translation type="unfinished"></translation>
     </message>
@@ -3563,7 +3589,12 @@
 <context>
     <name>QObject</name>
     <message>
-        <location filename="../main.cpp" line="88"/>
+        <location filename="../mainwindow.cpp" line="215"/>
+        <source>Error writing to pipe.</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../main.cpp" line="87"/>
         <source>Ooops in %1 at %2</source>
         <translation type="unfinished"></translation>
     </message>
@@ -3573,7 +3604,7 @@
         <translation type="unfinished"></translation>
     </message>
     <message>
-        <location filename="../mainwindow.cpp" line="181"/>
+        <location filename="../mainwindow.cpp" line="182"/>
         <source>Loading preset %1</source>
         <translation type="unfinished"></translation>
     </message>
@@ -3603,17 +3634,16 @@
         <translation type="unfinished"></translation>
     </message>
     <message>
-        <location filename="../modularsynth.cpp" line="1019"/>
         <source>Connection refused.</source>
         <translation type="obsolete">Keine Verbindung möglich.</translation>
     </message>
     <message>
-        <location filename="../modularsynth.cpp" line="1774"/>
+        <location filename="../modularsynth.cpp" line="1773"/>
         <source>No title for tab &apos;%1&apos; found.</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
-        <location filename="../modularsynth.cpp" line="1790"/>
+        <location filename="../modularsynth.cpp" line="1789"/>
         <source>No data for frame &apos;%1&apos; found.</source>
         <translation type="unfinished"></translation>
     </message>
@@ -3623,12 +3653,12 @@
         <translation type="unfinished"></translation>
     </message>
     <message>
-        <location filename="../modularsynth.cpp" line="1814"/>
+        <location filename="../modularsynth.cpp" line="1813"/>
         <source>No parameter name &apos;%1&apos; found.</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
-        <location filename="../modularsynth.cpp" line="1880"/>
+        <location filename="../modularsynth.cpp" line="1879"/>
         <source>No name for preset &apos;%1&apos; found.</source>
         <translation type="unfinished"></translation>
     </message>
@@ -3638,32 +3668,32 @@
         <translation type="unfinished"></translation>
     </message>
     <message>
-        <location filename="../synthdata.cpp" line="398"/>
+        <location filename="../synthdata.cpp" line="399"/>
         <source>Can&apos;t create ALSA thread</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
-        <location filename="../synthdata.cpp" line="407"/>
+        <location filename="../synthdata.cpp" line="408"/>
         <source>Closing ALSA...</source>
         <translation>Schließe ALSA-Verbindung...</translation>
     </message>
     <message>
-        <location filename="../synthdata.cpp" line="500"/>
+        <location filename="../synthdata.cpp" line="501"/>
         <source>Can&apos;t connect to JACK</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
-        <location filename="../synthdata.cpp" line="523"/>
+        <location filename="../synthdata.cpp" line="524"/>
         <source>Can&apos;t activate JACK</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
-        <location filename="../synthdata.cpp" line="535"/>
+        <location filename="../synthdata.cpp" line="536"/>
         <source>Closing JACK...</source>
         <translation>Schließe JACK-Verbindung...</translation>
     </message>
     <message>
-        <location filename="../main.cpp" line="65"/>
+        <location filename="../main.cpp" line="64"/>
         <source>Could not create ams home directory.</source>
         <translation type="unfinished"></translation>
     </message>
@@ -3673,17 +3703,17 @@
         <translation type="unfinished"></translation>
     </message>
     <message>
-        <location filename="../main.cpp" line="76"/>
+        <location filename="../main.cpp" line="75"/>
         <source>Failed to open file &apos;%1&apos;</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
-        <location filename="../mainwindow.cpp" line="194"/>
+        <location filename="../mainwindow.cpp" line="196"/>
         <source>Closing synthesizer...</source>
         <translation>Schließe Synthesizer...</translation>
     </message>
     <message>
-        <location filename="../mainwindow.cpp" line="200"/>
+        <location filename="../mainwindow.cpp" line="202"/>
         <source>Could not unlock preferences file.</source>
         <translation type="unfinished"></translation>
     </message>
diff -Naur ams-2.0.1/src/translations/ams_fr.ts ams/src/translations/ams_fr.ts
--- ams-2.0.1/src/translations/ams_fr.ts    2009-12-26 01:03:32.000000000 +0100
+++ ams/src/translations/ams_fr.ts  2009-12-28 18:10:58.000000000 +0100
@@ -1,5 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
-<!DOCTYPE TS><TS version="1.1" language="fr">
+<!DOCTYPE TS>
+<TS version="2.0" language="fr">
 <context>
     <name>ColorWidget</name>
     <message>
@@ -138,11 +139,15 @@
         <translation type="unfinished"></translation>
     </message>
     <message>
+        <location filename="../m_ad.cpp" line="74"/>
+        <location filename="../m_ad.cpp" line="172"/>
         <location filename="../m_ad.cpp" line="176"/>
         <source>Detune %1</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
+        <location filename="../m_ad.cpp" line="78"/>
+        <location filename="../m_ad.cpp" line="174"/>
         <location filename="../m_ad.cpp" line="178"/>
         <source>Drift %1</source>
         <translation type="unfinished"></translation>
@@ -334,6 +339,7 @@
         <translation type="unfinished"></translation>
     </message>
     <message>
+        <location filename="../m_advmcv.cpp" line="35"/>
         <location filename="../m_advmcv.cpp" line="61"/>
         <source>Controller %1</source>
         <translation type="unfinished"></translation>
@@ -416,6 +422,7 @@
 <context>
     <name>M_delay</name>
     <message>
+        <location filename="../m_delay.cpp" line="23"/>
         <location filename="../m_delay.cpp" line="33"/>
         <source>Delay</source>
         <translation type="unfinished"></translation>
@@ -866,7 +873,7 @@
         <translation type="unfinished"></translation>
     </message>
     <message>
-        <location filename="../m_mphlfo.cpp" line="49"/>
+        <location filename="../m_mphlfo.cpp" line="48"/>
         <source>Saw Down</source>
         <translation type="unfinished"></translation>
     </message>
@@ -1008,6 +1015,7 @@
         <translation type="unfinished"></translation>
     </message>
     <message>
+        <location filename="../m_pcmout.cpp" line="33"/>
         <location filename="../m_pcmout.cpp" line="53"/>
         <source>Keep</source>
         <translation type="unfinished"></translation>
@@ -1182,6 +1190,7 @@
         <translation type="unfinished"></translation>
     </message>
     <message>
+        <location filename="../m_scmcv.cpp" line="67"/>
         <location filename="../m_scmcv.cpp" line="176"/>
         <source>   Scale: </source>
         <translation type="unfinished"></translation>
@@ -1210,6 +1219,7 @@
 <context>
     <name>M_scope</name>
     <message>
+        <location filename="../m_scope.cpp" line="28"/>
         <location filename="../m_scope.cpp" line="49"/>
         <source>Scope</source>
         <translation type="unfinished"></translation>
@@ -1318,6 +1328,7 @@
         <translation type="unfinished"></translation>
     </message>
     <message>
+        <location filename="../m_scquantizer.cpp" line="51"/>
         <location filename="../m_scquantizer.cpp" line="204"/>
         <source>   Scale: </source>
         <translation type="unfinished"></translation>
@@ -1341,6 +1352,7 @@
         <translation type="unfinished"></translation>
     </message>
     <message>
+        <location filename="../m_seq.cpp" line="33"/>
         <location filename="../m_seq.cpp" line="38"/>
         <source>Trigger</source>
         <translation type="unfinished"></translation>
@@ -1430,16 +1442,19 @@
 <context>
     <name>M_spectrum</name>
     <message>
+        <location filename="../m_spectrum.cpp" line="7"/>
         <location filename="../m_spectrum.cpp" line="88"/>
         <source>Spectrum</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
+        <location filename="../m_spectrum.cpp" line="13"/>
         <location filename="../m_spectrum.cpp" line="60"/>
         <source>In 0</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
+        <location filename="../m_spectrum.cpp" line="14"/>
         <location filename="../m_spectrum.cpp" line="61"/>
         <source>In 1</source>
         <translation type="unfinished"></translation>
@@ -1573,7 +1588,7 @@
 <context>
     <name>M_stereomix</name>
     <message>
-        <location filename="../m_stereomix.cpp" line="107"/>
+        <location filename="../m_stereomix.cpp" line="106"/>
         <source>Stereo Mixer %1</source>
         <translation type="unfinished"></translation>
     </message>
@@ -1626,16 +1641,19 @@
         <translation type="unfinished"></translation>
     </message>
     <message>
+        <location filename="../m_vca.cpp" line="33"/>
         <location filename="../m_vca.cpp" line="40"/>
         <source>Gain 1</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
+        <location filename="../m_vca.cpp" line="34"/>
         <location filename="../m_vca.cpp" line="41"/>
         <source>In 0</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
+        <location filename="../m_vca.cpp" line="35"/>
         <location filename="../m_vca.cpp" line="42"/>
         <source>In 1</source>
         <translation type="unfinished"></translation>
@@ -1968,6 +1986,7 @@
         <translation type="unfinished"></translation>
     </message>
     <message>
+        <location filename="../m_vcf.cpp" line="37"/>
         <location filename="../m_vcf.cpp" line="56"/>
         <source>Resonance</source>
         <translation type="unfinished"></translation>
@@ -2071,6 +2090,7 @@
         <translation type="unfinished"></translation>
     </message>
     <message>
+        <location filename="../m_vco.cpp" line="44"/>
         <location filename="../m_vco.cpp" line="61"/>
         <source>PW</source>
         <translation type="unfinished"></translation>
@@ -2086,6 +2106,7 @@
         <translation type="unfinished"></translation>
     </message>
     <message>
+        <location filename="../m_vco.cpp" line="49"/>
         <location filename="../m_vco.cpp" line="67"/>
         <source>Saw</source>
         <translation type="unfinished"></translation>
@@ -2194,6 +2215,7 @@
         <translation type="unfinished"></translation>
     </message>
     <message>
+        <location filename="../m_vco2.cpp" line="41"/>
         <location filename="../m_vco2.cpp" line="69"/>
         <source>PW</source>
         <translation type="unfinished"></translation>
@@ -2382,6 +2404,8 @@
         <translation type="unfinished"></translation>
     </message>
     <message>
+        <location filename="../m_vcorgan.cpp" line="59"/>
+        <location filename="../m_vcorgan.cpp" line="66"/>
         <location filename="../m_vcorgan.cpp" line="68"/>
         <source>Volume %1</source>
         <translation type="unfinished"></translation>
@@ -2644,17 +2668,17 @@
         <translation type="unfinished"></translation>
     </message>
     <message>
-        <location filename="../m_wavout.cpp" line="241"/>
+        <location filename="../m_wavout.cpp" line="246"/>
         <source>Choose Wave File</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
-        <location filename="../m_wavout.cpp" line="242"/>
+        <location filename="../m_wavout.cpp" line="247"/>
         <source>WAV files (*.wav)</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
-        <location filename="../m_wavout.cpp" line="272"/>
+        <location filename="../m_wavout.cpp" line="277"/>
         <source>Time: %1:%2:%3 </source>
         <translation type="unfinished"></translation>
     </message>
@@ -2687,6 +2711,7 @@
         <translation>Ai&amp;de</translation>
     </message>
     <message>
+        <location filename="../mainwindow.cpp" line="49"/>
         <location filename="../mainwindow.cpp" line="52"/>
         <source>&amp;New</source>
         <translation>&amp;Nouveau</translation>
@@ -3072,74 +3097,75 @@
         <translation>A propos de &amp;Qt...</translation>
     </message>
     <message>
-        <location filename="../mainwindow.cpp" line="231"/>
+        <location filename="../mainwindow.cpp" line="239"/>
         <source>Unnamed file was changed.
 Save changes?</source>
         <translation>Le fichier sans nom a été modifié. Enregistrer les modifications?</translation>
     </message>
     <message>
-        <location filename="../mainwindow.cpp" line="234"/>
+        <location filename="../mainwindow.cpp" line="241"/>
         <source>File &apos;%1&apos; was changed.
 Save changes?</source>
         <translation>Le fichier &apos;%1&apos; a été modifié. Enregistrer les modifications?</translation>
     </message>
     <message>
-        <location filename="../mainwindow.cpp" line="236"/>
+        <location filename="../mainwindow.cpp" line="244"/>
         <source>Save changes</source>
         <translation>Enregistrer les modifications</translation>
     </message>
     <message>
-        <location filename="../mainwindow.cpp" line="237"/>
+        <location filename="../mainwindow.cpp" line="245"/>
         <source>&amp;Yes</source>
         <translation>&amp;Oui</translation>
     </message>
     <message>
-        <location filename="../mainwindow.cpp" line="237"/>
+        <location filename="../mainwindow.cpp" line="245"/>
         <source>&amp;No</source>
         <translation>&amp;Non</translation>
     </message>
     <message>
-        <location filename="../mainwindow.cpp" line="237"/>
+        <location filename="../mainwindow.cpp" line="245"/>
         <source>Cancel</source>
         <translation>Annuler</translation>
     </message>
     <message>
-        <location filename="../mainwindow.cpp" line="244"/>
+        <location filename="../mainwindow.cpp" line="252"/>
         <source>Open patch file</source>
         <translation>Ouvrir fichier patch</translation>
     </message>
     <message>
-        <location filename="../mainwindow.cpp" line="355"/>
+        <location filename="../mainwindow.cpp" line="253"/>
+        <location filename="../mainwindow.cpp" line="365"/>
         <source>Patch files</source>
         <translation>Fichiers patch</translation>
     </message>
     <message>
-        <location filename="../mainwindow.cpp" line="306"/>
+        <location filename="../mainwindow.cpp" line="314"/>
         <source>Could not read file &apos;%1&apos;</source>
         <translation>Echec de lecture du fichier &apos;%1&apos;</translation>
     </message>
     <message>
-        <location filename="../mainwindow.cpp" line="337"/>
+        <location filename="../mainwindow.cpp" line="346"/>
         <source>Could not open file &apos;%1&apos;</source>
         <translation>Echec d&apos;ouverture du fichier &apos;%1&apos;</translation>
     </message>
     <message>
-        <location filename="../mainwindow.cpp" line="354"/>
+        <location filename="../mainwindow.cpp" line="364"/>
         <source>Save patch file</source>
         <translation>Enregistrer fichier patch</translation>
     </message>
     <message>
-        <location filename="../mainwindow.cpp" line="368"/>
+        <location filename="../mainwindow.cpp" line="378"/>
         <source>Saving aborted</source>
         <translation>Enregistrement annulé</translation>
     </message>
     <message>
-        <location filename="../mainwindow.cpp" line="377"/>
+        <location filename="../mainwindow.cpp" line="387"/>
         <source>noname</source>
         <translation>sansnom</translation>
     </message>
     <message>
-        <location filename="../mainwindow.cpp" line="393"/>
+        <location filename="../mainwindow.cpp" line="403"/>
         <source>About Qt</source>
         <translation>A propos de Qt</translation>
     </message>
@@ -3147,12 +3173,11 @@
 <context>
     <name>MidiControllerModel</name>
     <message>
-        <location filename="../midiwidget.cpp" line="65"/>
         <source>type: %1 channel: %2 param: %3</source>
         <translation type="obsolete">type: %1 canal: %2 param: %3</translation>
     </message>
     <message>
-        <location filename="../midiwidget.cpp" line="78"/>
+        <location filename="../midiwidget.cpp" line="77"/>
         <source>Module</source>
         <translation>Module</translation>
     </message>
@@ -3162,7 +3187,7 @@
         <translation>Controleur MIDI / Paramètre</translation>
     </message>
     <message>
-        <location filename="../midiwidget.cpp" line="65"/>
+        <location filename="../midiwidget.cpp" line="64"/>
         <source>Type: %1 Channel: %2 Param: %3</source>
         <translation type="unfinished"></translation>
     </message>
@@ -3240,7 +3265,7 @@
         <translation>&amp;Inverser Signe MIDI</translation>
     </message>
     <message>
-        <location filename="../midiwidget.cpp" line="512"/>
+        <location filename="../midiwidget.cpp" line="511"/>
         <source>This will erase all presets for this configuration. Continue?</source>
         <translation>Ceci effacera tous les presets pour cette configuration. Continuer?</translation>
     </message>
@@ -3298,7 +3323,6 @@
         <translation>Charger Couleurs</translation>
     </message>
     <message>
-        <location filename="../modularsynth.cpp" line="1168"/>
         <source>AlsaModularSynth Color files (*.acs)</source>
         <translation type="obsolete">Fichiers Couleurs AlsaModularSynth (*.acs)</translation>
     </message>
@@ -3318,27 +3342,28 @@
         <translation>Echec d&apos;enregistrement fichier.</translation>
     </message>
     <message>
-        <location filename="../modularsynth.cpp" line="1265"/>
+        <location filename="../modularsynth.cpp" line="1263"/>
         <source>Could not find LADSPA plugin &quot;%1&quot; from library &quot;%2&quot;.</source>
         <translation>Plugin LADSPA &quot;%1&quot; de la librairie &quot;%2&quot; non trouvé.</translation>
     </message>
     <message>
-        <location filename="../modularsynth.cpp" line="1482"/>
+        <location filename="../modularsynth.cpp" line="1480"/>
         <source>Input port %1 of module %2 is already connected. New connection to module %3 ignored.</source>
         <translation>Le port d&apos;entrée %1 du module %2 est déjà connecté. Nouvelle connexion au module %3 ignorée.</translation>
     </message>
     <message>
-        <location filename="../modularsynth.cpp" line="1738"/>
+        <location filename="../modularsynth.cpp" line="1737"/>
         <source>Unknown MIDI controller tag found: %1</source>
         <translation>Tag controleur MIDI inconnu détecté: %1</translation>
     </message>
     <message>
-        <location filename="../modularsynth.cpp" line="1258"/>
+        <location filename="../modularsynth.cpp" line="1257"/>
         <source>Loading LADSPA plugin &quot;%1&quot; from library &quot;%2&quot;.</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
-        <location filename="../modularsynth.cpp" line="1168"/>
+        <location filename="../modularsynth.cpp" line="1125"/>
+        <location filename="../modularsynth.cpp" line="1167"/>
         <source>AlsaModularSynth Color files</source>
         <translation type="unfinished"></translation>
     </message>
@@ -3566,17 +3591,22 @@
 <context>
     <name>QObject</name>
     <message>
-        <location filename="../main.cpp" line="65"/>
+        <location filename="../mainwindow.cpp" line="215"/>
+        <source>Error writing to pipe.</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../main.cpp" line="64"/>
         <source>Could not create ams home directory.</source>
         <translation>Echec de création du repertoire ams home.</translation>
     </message>
     <message>
-        <location filename="../main.cpp" line="76"/>
+        <location filename="../main.cpp" line="75"/>
         <source>Failed to open file &apos;%1&apos;</source>
         <translation>Echec d&apos;ouverture du fichier &apos;%1&apos;</translation>
     </message>
     <message>
-        <location filename="../main.cpp" line="88"/>
+        <location filename="../main.cpp" line="87"/>
         <source>Ooops in %1 at %2</source>
         <translation>Ooops dans %1 à %2</translation>
     </message>
@@ -3591,17 +3621,17 @@
         <translation>Chemin des Presets: %1</translation>
     </message>
     <message>
-        <location filename="../mainwindow.cpp" line="181"/>
+        <location filename="../mainwindow.cpp" line="182"/>
         <source>Loading preset %1</source>
         <translation>Chargement du Preset %1</translation>
     </message>
     <message>
-        <location filename="../mainwindow.cpp" line="194"/>
+        <location filename="../mainwindow.cpp" line="196"/>
         <source>Closing synthesizer...</source>
         <translation>Fermeture du synthétiseur...</translation>
     </message>
     <message>
-        <location filename="../mainwindow.cpp" line="200"/>
+        <location filename="../mainwindow.cpp" line="202"/>
         <source>Could not unlock preferences file.</source>
         <translation>Echec de déverouillage du fichier de préférences.</translation>
     </message>
@@ -3636,12 +3666,12 @@
         <translation>Tous les ports d&apos;entrée disponibles sont déjà utilisés</translation>
     </message>
     <message>
-        <location filename="../modularsynth.cpp" line="1774"/>
+        <location filename="../modularsynth.cpp" line="1773"/>
         <source>No title for tab &apos;%1&apos; found.</source>
         <translation>Titre pour l&apos;onglet &apos;%1&apos; introuvable.</translation>
     </message>
     <message>
-        <location filename="../modularsynth.cpp" line="1790"/>
+        <location filename="../modularsynth.cpp" line="1789"/>
         <source>No data for frame &apos;%1&apos; found.</source>
         <translation>Données pour le cadre &apos;%1&apos; non trouvées.</translation>
     </message>
@@ -3651,12 +3681,12 @@
         <translation>Valeurs de paramètre introuvables.</translation>
     </message>
     <message>
-        <location filename="../modularsynth.cpp" line="1814"/>
+        <location filename="../modularsynth.cpp" line="1813"/>
         <source>No parameter name &apos;%1&apos; found.</source>
         <translation>Nom du paramètre &apos;%1&apos; non trouvé.</translation>
     </message>
     <message>
-        <location filename="../modularsynth.cpp" line="1880"/>
+        <location filename="../modularsynth.cpp" line="1879"/>
         <source>No name for preset &apos;%1&apos; found.</source>
         <translation>Nom du Preset &apos;%1&apos; non trouvé.</translation>
     </message>
@@ -3666,27 +3696,27 @@
         <translation>Echec d&apos;établir un fil ALSA avec priorité RT</translation>
     </message>
     <message>
-        <location filename="../synthdata.cpp" line="398"/>
+        <location filename="../synthdata.cpp" line="399"/>
         <source>Can&apos;t create ALSA thread</source>
         <translation>Echec d&apos;établir un fil ALSA</translation>
     </message>
     <message>
-        <location filename="../synthdata.cpp" line="407"/>
+        <location filename="../synthdata.cpp" line="408"/>
         <source>Closing ALSA...</source>
         <translation>Fermeture d&apos;ALSA...</translation>
     </message>
     <message>
-        <location filename="../synthdata.cpp" line="500"/>
+        <location filename="../synthdata.cpp" line="501"/>
         <source>Can&apos;t connect to JACK</source>
         <translation>Echec de connexion à JACK</translation>
     </message>
     <message>
-        <location filename="../synthdata.cpp" line="523"/>
+        <location filename="../synthdata.cpp" line="524"/>
         <source>Can&apos;t activate JACK</source>
         <translation>Echec d&apos;activation de JACK</translation>
     </message>
     <message>
-        <location filename="../synthdata.cpp" line="535"/>
+        <location filename="../synthdata.cpp" line="536"/>
         <source>Closing JACK...</source>
         <translation>Fermeture de JACK...</translation>
     </message>
     
